openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -595,6 +626,32 @@ def build_anim(project_name, asset_name):
     load_subset(project_name, board_repre, "Background")
 
 
+def build_lipsync(project_name: str, shot_name: str):
+    """Build lipsync workfile.
+
+    Args:
+        project_name (str):  Current project name from OpenPype Session.
+        shot_name (str):  Current shot name from OpenPype Session.
+    """
+
+    representations = download_kitsu_casting(
+        project_name, shot_name, rig_only=True
+    )
+
+    for representation in representations:
+        try:
+            load_subset(
+                project_name,
+                representation,
+                "rigMain",
+                "LinkRigLoader",
+            )
+        except TypeError:
+            print(
+                f"Can't load {representation['context']['asset']} {'rigMain'}."
+            )
```

Comment:
`TypeError` can only occur if `representation is None`. Therefore, you better test that representation is not None and don't overload with a `try`.